examples/process-quarkus-example-persistence.yaml

```diff
@@ -1,3 +1,24 @@
+---
+#Strimzi operator should be pre-installed in namespace
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoInfra
+metadata:
+  name: kogito-kafka-infra
+spec:
+  resource:
+    apiVersion: kafka.strimzi.io/v1beta1
+    kind: Kafka
```

Comment:
If the name is not set we assume `KogitoInfra`'s name?

Reply:
if name is not provided under resource then for infinispan we assume **kogito-infinispan** and for kafka we assume **kogito-kafka**

Reply:
We might need to add this in the docs since it´s not clear from the interface.